db/migrate_cards/20180105134605_created_at_for_answers.rb

```diff
@@ -0,0 +1,19 @@
+# -*- encoding : utf-8 -*-
+
+class CreatedAtForAnswers < Card::Migration
+  def up
+    Answer.where(created_at: nil).each do |a|
+      next unless a.card
+      if a.card.type_id == Card::MetricValueID
+        if a.card.researched?
+          a.update_attributes! created_at: a.card.created_at
+        else
+          a.update_attributes! created_at: a.card.created_at, answer_id: nil
+          a.card.delete!
```

Comment:
sigh, I keep losing this comment because of tweaks:

this `delete!` should not cause the answer lookup record to be deleted, because that event uses the answer_id to find the answer (and the answer_id is now nil).

Reply:
so I think this addresses all four of your (valid) concerns.